src/plugins/search/component.js

```diff
@@ -171,11 +171,11 @@ function bindEvents() {
 
   let timeId;
   // Prevent to Fold sidebar
-  Docsify.dom.on(
-    $search,
-    'click',
-    e => e.target.tagName !== 'A' && e.stopPropagation()
-  );
+//   Docsify.dom.on(
+//     $search,
+//     'click',
+//     e => e.target.tagName !== 'A' && e.stopPropagation()
+//   );
```

Comment:
Why this :?

Reply:
fix this bug
https://github.com/docsifyjs/docsify/issues/1098

 in most cases, the element that triggers the click event is not tag A .
At this time, stopPropagation event causing click the search results on the left, the right page cannot be located.
In your source code, the location is triggered by listening to the click event

Reply:
Oh I see. I wonder why that was there though. Why did it wasn't too prevent clicks on non-anchor tags?